### ISSUE TITLE
Fix `svg-img-alt` violation 

### ISSUE THREAD

**Issue by DanielCliftonGuardian** [guardian/dotcom-rendering#15069]
WCAG: 2.0 A (1.1.1)
Affected: 1 story
Fix: Add <title> or aria-label to SVG elements with role="img"

Affected Components
Components/Standfirst (1 story)
